src/server.js

```diff
@@ -248,8 +243,7 @@ function init(socket, client) {
 								const obj = {};
 
 								if (success) {
-									obj.success = "Successfully updated your password, all your other sessions were logged out";
-									obj.token = client.config.token;
+									obj.success = "Successfully updated your password";
```

Comment:
It's pretty silly to log everything out on password change. Push PR will also store push endpoint config in the session object, so resetting all sessions would break push notifications.